src/main/java/com/example/demo/EmailConfig.java

```diff
@@ -0,0 +1,48 @@
+package com.example.demo;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import java.util.Properties;
+
+@EnableAutoConfiguration
+@Configuration
+public class EmailConfig extends DemoApplication{
+
+    @Bean
+    public SimpleMailMessage getMessage() {
+        return new SimpleMailMessage();
+    }
+
+    @Bean
+    public JavaMailSender getJavaMailSender() {
+        JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
+        mailSender.setHost("smtp.gmail.com");
+        mailSender.setPort(587);
+
+        mailSender.setUsername(Constants.MY_EMAIL);
```

Comment:
This shouldn't be a constant. Consider moving it into application properties. If you want to run this application for different sender email you will not able to do so if you go with constants. However, it's easy to do with `application.properties`